universe/Effect.h

```diff
@@ -1123,7 +1123,7 @@ class FO_COMMON_API GenerateSitRepMessage : public EffectBase {
     const std::string& Icon() const
     { return m_icon; }
 
-    const auto& MessageParameters() const
+    auto MessageParameters() -> const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*>>&
     { return m_message_parameters; }
```

Comment:
My original comment was to point out that the single line of code was not C++11 and would not compile with the `--std=c++11` switch in either clang/gcc.  I was unaware that VC++ did not have a switch to toggle standards and assumed that you had forgotten to change back to C++11 mode.

I checked yesterday and the codebase compiles and runs under C++14 for clang and gcc on linux.  This PR did not successfully compile on either.

If we try the C++ version change on OSX and it works, then I suggest that we upgrade to C++14.  It will make the behavior of VC++ and the the other two compilers more similar.